exercises/Preprocessing/notebooks/00_Loading_and_preprocessing_scRNAseq_data.ipynb

```diff
@@ -1110,7 +1110,9 @@
    "source": [
     "## 4. What happens if we don't preprocess correctly?\n",
     "\n",
-    "We can compare the results of preprocessing by examining the data after dimensionality reduction (e.g. PCA, t-SNE, UMAP, PHATE). We'll learn more about these methods later, but for now, let's see what trends we can see in the data with and without correct preprocessing."
+    "We can compare the results of preprocessing by examining the data after dimensionality reduction (e.g. PCA, t-SNE, UMAP, PHATE). We'll learn more about these methods later, but for now, let's see what trends we can see in the data with and without correct preprocessing.\n",
+    "\n",
+    "> Do we need to restart the environment after installing phate?"
```

Comment:
This shouldn't be necessary. In fact, recent changes in Colab mean we shouldn't even have to restart after installing `scprep`.